wren-ui/src/apollo/server/adaptors/wrenEngineAdaptor.ts

```diff
@@ -47,6 +47,25 @@ export interface DescribeStatementResponse {
   columns: ColumnMetadata[];
 }
 
+export enum WrenEngineValidateStatusEnum {
```

Comment:
nit: feel like `Enum` postfix is unnecessary. Naming could be `WrenEngineValidationStatus`.

Reply:
delivered in 9ec6f9620239b869a9716daf8c6827d85c23995a